crates/iota-indexer/src/handlers/committer.rs

```diff
@@ -175,6 +175,15 @@ async fn commit_checkpoints<S>(
             })
             .expect("Advancing epochs in DB should not fail.");
         metrics.total_epoch_committed.inc();
+
+        // Refresh participation metrics after advancing epoch
+        state
+            .refresh_participation_metrics()
+            .await
+            .tap_err(|e| {
+                error!("Failed to update participation metrics: {}", e.to_string());
```

Comment:
.to_string() not needed, as this should be just Display? Haven't checked the error type
```suggestion
                error!("Failed to update participation metrics: {e}");
```